app/uk/gov/hmrc/merchandiseinbaggage/model/core/Declaration.scala

```diff
@@ -20,6 +20,11 @@ object TraderName {
 case class AmountInPence(value: Double)
 object AmountInPence {
   implicit val format: Format[AmountInPence] = ValueClassFormat.formatDouble(value => AmountInPence.apply(value))(_.value)
+
+  implicit class Rounding(amount: Double) {
```

Comment:
Do we need rounding to two decimal places? We are getting the amount in pence, so in order for us to get it to Pounds and pence we would just divide by 100. We don't want to store AmountInPence as a floating point number, it should be Integer or Long

Reply:
like this???  implicit class Rounding(amount: Double) {
    def twoDecimalsHalfUp: AmountInPence = AmountInPence(amount / 100.toDouble)
  }